README.rst

```diff
@@ -96,7 +96,9 @@ Background and Documentation
 NiFi Version Support
 --------------------
 
```

Comment:
I think that if our answer to issues with 2.0.0-MX is going to be "we will fix that when we do the refactoring for the new branch" we should explicitly state that here and set expectations.

Reply:
How about this language:


| Currently we are testing against NiFi versions 1.9.2 - 1.27.0, and NiFi-Registry versions 0.3.0 - 1.27.0.

| We have also tested against the latest 2.0.0-M4 release candidates using the 1.x SDK and have found that basic functionality works as expected.
| Apache NiFi offers no compatibility guarantees between major versions, and while many functions may work the same, you should test carefully for your specific use case.
| In future we will create a specific branch of NiPyAPI for NiFi 2.x SDKs, and maintain separate NiFi 1.x and NiFi 2.x clients.

| If you find a version compatibility problem please raise an `issue <https://github.com/Chaffelson/nipyapi/issues>`_

Reply:
yes, that is great